buildsystem/compdeps/sun.misc/META-INF/MANIFEST.MF

```diff
@@ -1,7 +1,7 @@
 Manifest-Version: 1.0
 Bundle-Version: 1.0.5.qualifier
-Export-Package: sun.misc;sun.reflect
+Export-Package: sun.misc;sun.reflect;com.sun.activation.registries;com.sun.source.tree;com.sun.source.util
```

Comment:
If com.sun.activation:jakarta.activation is used then this change is not needed

Reply:
_jakarta.activation_ dependency changed. _jakarta.activation_ is used there instead of _jakarta.activation-api_ . In case of _jakarta.activation-api_ Tycho compiler throws following error. `Missing requirement: jakarta.activation-api 1.2.1 requires 'java.package; com.sun.activation.registries 0.0.0' but it could not be found`.

Reply:
https://github.com/eclipse-ee4j/jaf/issues/17 is filed for the API